src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -171,7 +171,7 @@ export class ValueElementRenderer {
   }
 
   private buildListItem() {
-    this.listItem = $$('li', { className: 'coveo-facet-value coveo-facet-selectable' }).el;
+    this.listItem = $$('li', { className: 'coveo-facet-value coveo-facet-selectable', ariaLabel: this.ariaLabel }).el;
```

Comment:
This allows VoiceOver and TalkBack to read out loud what search result was focused. However, this also adds a redundant label to facet values outside of search results, which already have this label in one of their children.